src/app/issuer/components/badgeclass-edit-form/badgeclass-edit-form.component.ts

```diff
@@ -461,6 +463,11 @@ export class BadgeClassEditFormComponent extends BaseAuthenticatedRoutableCompon
 
 		if (newTag.length > 0) {
 			this.tags.add(newTag);
+			if (!this.existingTags.some((tag) => tag.name === newTag)) {
+				this.existingTags.push({ id: this.existingTags.length, name: String(newTag) });
```

Comment:
@jona159 you didn't address this one yet, right?

Reply:
Since i did not find a state in the history where the autocomplete automatically added the tag to the existingTags, i resorted to updating the array manually. Would you suggest something else?

Reply:
The `this.existingTags` collection is only used for the autocomplete though, right? So by adding the tags here, the only thing you achieve is that they also appear in the autocomplete, which is useless, if they're already added (why would you want to add a tag twice?). If you *don't* do this however, this also solves the other issue of the tag not being *removed* from the autocomplete, because you never *added* it.

Reply:
Ok got it now... Removed this part